app/javascript/controllers/persisted_form_controller.ts

```diff
@@ -0,0 +1,42 @@
+import { ApplicationController } from './application_controller';
+
+export class PersistedFormController extends ApplicationController {
+  static values = {
+    key: String
+  };
+  static targets = ['input'];
+
+  declare readonly keyValue: string;
+  declare readonly inputTarget: HTMLInputElement;
+
+  connect() {
+    this.on('submit', () => {
```

Comment:
J'ai du mal avec la différence de déclaration entre `onInput` et le `this.on('submit')`. J'ai l'impression qu'on voudrait que les deux évènements soient déclarés de la même manière.